app/frontend/packs/sort-by-filter.js

```diff
@@ -0,0 +1,10 @@
+const sortByFilter = () => {
+    const filter = document.querySelector('[data-module="sort-by-filter-option"]')
+    if(filter){
+        filter.addEventListener('change', (event) => {
+            event.target.form.submit();
+        });
+    };
+};
```

Comment:
This makes sense to me. The only question I've got is whether we need to add the `data-sort-by-filter-option`? Can we use `#sortby-field` as the selector? @paulrobertlloyd might have a view on a preferred way?

Reply:
I’d usually select a field via a data attribute, or a `js-` prefixed class.